CWE/history/countries/CHI - China.txt

```diff
@@ -4,15 +4,18 @@ culture = chinese_hokkien
 culture = chinese_hakka
 culture = manchu
 culture = miao
-culture = hani
 culture = hui
 culture = yi
-culture = min
-culture = yue
+culture = bai
+culture = dong
+culture = tujia
 culture = zhuang
 culture = chinese_cantonese
 culture = mongol
 culture = taiwanese
+culture = korean
+culture = asian_minor
```

Comment:
Asian minor shouldn't be added as they represent an assortment of small cultures that might not be Chinese related.